### ISSUE TITLE
RnD really needs to change

### ISSUE THREAD

**Issue by SGuivenne** [d3athrow/vgstation13#33011]
#### Description of issue

Space Station 13 is a game fun for its diversity.  One round can be absolute chaos of trying to hold an imploding station together and another round can be building a new escape lounge.  These differences are what makes the game worthwhile.  RnD does nothing for this in its current iteration and has, at this point, become a checklist so bog standard that you could likely program a macro to just do it for you and nobody would even know.  You literally follow the same list of items to breakdown, calculated to the fastest possible route to RnD everything you’d want in a round, or you merely RP that you’re doing your job as mining breaks in and does it for you.  This isn’t fun and it contributes nothing of value to any round whatsoever.  It doesn’t even require intelligence. 

With the engines at least you have options and creative differences along side the easy ways and you can really switch it up if you want to.  You don’t get that here.

Tldr: RnD in its current iteration is shit, is basically worthless, and I implore you guys to consider changing it up or considering an overhaul of it.  

[featurerequest]

**Comment by WindowsErrors** [comment]
ok but how

**Comment by Eneocho** [comment]
>calculated to the fastest possible route to RnD everything you’d want in a round

Not sure about that. The wiki guides have not been reworked to consider the whole ton of new objects added since.
I get it though, current RnD is garbage and I wish we had a different system.

>ok but how

Bring back T.E.T.R.I.S. RnD (#18555) as an alternative. You do the gay and boring regular research or do the gentleman's research by playing betris with your fellow scientists.

**Comment by stormcloud23** [comment]
seconding bringing tetris back

**Comment by Inorien** [comment]
just spitballing an idea, how about making it more like real science:

>how do we make better power cell?
>power cell has a couple of core "techs" involved with it
> two variables to improve - capacity and charge rate? or just make them vague intrinsic properties?
> science needs to perform experiments (not unlike botany in spirit) to figure out ideal values for certain variables (think of tweaking the gas fuel mix to the perfect ratio for a TEG)
> print power cell with some test parameters, throw in analysis machine, get performance metrics of the battery
> refine guess on parameters, retest, repeat
> values closer to the hidden "correct" value (randomised between rounds) yields better results on power cells printed at the protolathe
> eg. basic power cell can be (roll?) between 500-1000 capacity, more advanced models raise the maximum to 1500, 2000, etc
> next tier up adds 1? 2? extra variables to tune
> you can always try making the next tier of power cell but it'll be garbage without good research on the lower tiers
> eg. perfect basic power cell research to 1000 capacity, make the next tier up, depending on your initial guesses for new variable(s) you might only make something with 1054 capacity
> repeat experiments on this to tune the new variable to eventually make 1500 capacity cells

this will keep science busy throughout the round and will force focus on improving specific techs, rather than running through a checklist for a few minutes and having everything before even getting materials. it also decouples science a bit from stuff that's already in the map and opens the door to great rewards after enough focused effort is channeled into it, as well as being more generally interactive
the downside is pretty obvious, it requires more work to get to the fun toys so a balance would need to be found between tedium and reward
